src/Subscribers/DoctrineEncryptSubscriber.php

```diff
@@ -175,19 +167,14 @@ public function postLoad(LifecycleEventArgs $args)
      */
     public function getSubscribedEvents()
```

Comment:
it has one, no? https://github.com/BeatSwitch/doctrine-encrypt/blob/7d06c466a6dc9d29ae3ecf488b03b7b8e5b2ca5c/src/Subscribers/DoctrineEncryptSubscriber.php#L166

Cause 5.4 and up are supported, can't just do `: array`
https://github.com/51systems/doctrine-encrypt/blob/master/.travis.yml#L4